packages/store/addon/-private/system/core-store.ts

```diff
@@ -93,6 +93,7 @@ type SchemaDefinitionService = import('../ts-interfaces/schema-definition-servic
 type PrivateSnapshot = import('./snapshot').PrivateSnapshot;
 type Relationship = import('@ember-data/record-data/-private').Relationship;
 type RecordDataClass = typeof import('@ember-data/record-data/-private').RecordData;
+type RequestCache = import('./request-cache').default;
```

Comment:
Note for future fun: you *should* now be able to switch all of these kinds of imports over to the `import type` syntax:

```ts
import type RequestCache from './request-cache';
```

(I'd love to hear if that *doesn't* work with your current tooling configuration for some reason!)

Reply:
I've been considering making the migration, we have a custom lint tule for type only imports that I'd want to update but it is a nicer import syntax :)   Does it also allow you to import types for multiple named declarations in one go?

Reply:
Yep, it is perfectly symmetrical with regular import statements! So very much nicer that way.￼